### ISSUE TITLE
InvalidInput ODATA Error: One of the request inputs is not valid.

### ISSUE THREAD

**Issue by sheeeng** [microsoft/AzureStorageExplorer#5434]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.22.1

### Architecture

x64

### Storage Explorer Build Number

20220122.3

### Platform

macOS

### OS Version

macOS 12.1 (21C52)

### Bug Description

The below information taken from Microsoft Azure Storage Explorer.

```text
Version: 1.22.1
AzCopy Version: 10.13.0

Platform: macOS
Architecture: x64 
Build Number: 20220122.3 
Commit: 7b2b13504 
Support ID: 07aad20f-8499-adec-9634-af7ee7c907e7 

Copyright © 2022 Microsoft. All rights reserved.
```

These error message was displayed when I viewed the Azure Table storage inside Microsoft Azure Storage Explorer.

```json
{"odata.error":{"code":"InvalidInput","message":{"lang":"en-US","value":"One of the request inputs is not valid.\nRequestId:33fafa70-7002-008d-216b-1f2076000000\nTime:2022-02-11T17:17:48.9518298Z"}}}
```

```json
{
  "name": "RestError",
  "message": "{\"odata.error\":{\"code\":\"InvalidInput\",\"message\":{\"lang\":\"en-US\",\"value\":\"One of the request inputs is not valid.\\nRequestId:3a117790-e002-007b-4768-1f5538000000\\nTime:2022-02-11T16:56:50.0462714Z\"}}}",
  "stack": "RestError: {\"odata.error\":{\"code\":\"InvalidInput\",\"message\":{\"lang\":\"en-US\",\"value\":\"One of the request inputs is not valid.\\nRequestId:3a117790-e002-007b-4768-1f5538000000\\nTime:2022-02-11T16:56:50.0462714Z\"}}}\n    at handleErrorResponse (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/@azure/core-client/dist/index.js:1535:19)\n    at deserializeResponseBody (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/@azure/core-client/dist/index.js:1470:45)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)",
  "statusCode": 400,
  "request": {
    "url": "https://REDACTED_STORAGE_ACCOUINT_NAME.table.core.windows.net/CabTable()?NextPartitionKey=1!12!Q2FiUXVlcnk-&NextRowKey=",
    "headers": {
      "accept": "application/json;odata=minimalmetadata",
      "x-ms-version": "2019-02-02",
      "dataserviceversion": "3.0",
      "accept-encoding": "gzip,deflate",
      "user-agent": "Microsoft Azure Storage Explorer, 1.22.1, darwin azsdk-js-data-tables/13.0.1 core-rest-pipeline/1.3.3 Node/v14.16.0 OS/(x64-Darwin-21.2.0)",
      "x-ms-client-request-id": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
      "x-ms-date": "Fri, 11 Feb 2022 16:56:49 GMT",
      "authorization": "SharedKeyLite REDACTED_STORAGE_ACCOUINT_NAME:ut2uNqtJMqXoQVIZG8pqyrnTUleLsGaSqgAnRAtgc0Q="
    },
    "method": "GET",
    "timeout": 0,
    "disableKeepAlive": false,
    "streamResponseStatusCodes": {},
    "withCredentials": false,
    "tracingOptions": {
      "spanOptions": {},
      "tracingContext": {
        "_currentContext": {}
      }
    },
    "requestId": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
    "allowInsecureConnection": true
  },
  "response": {
    "status": 400,
    "headers": {
      "content-length": "199",
      "content-type": "application/json",
      "server": "Windows-Azure-Table/1.0 Microsoft-HTTPAPI/2.0",
      "x-ms-request-id": "3a117790-e002-007b-4768-1f5538000000",
      "x-ms-client-request-id": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
      "x-ms-version": "2019-02-02",
      "x-ms-error-code": "InvalidInput",
      "date": "Fri, 11 Feb 2022 16:56:49 GMT"
    },
    "request": {
      "url": "https://REDACTED_STORAGE_ACCOUINT_NAME.table.core.windows.net/CabTable()?NextPartitionKey=1!12!Q2FiUXVlcnk-&NextRowKey=",
      "headers": {
        "accept": "application/json;odata=minimalmetadata",
        "x-ms-version": "2019-02-02",
        "dataserviceversion": "3.0",
        "accept-encoding": "gzip,deflate",
        "user-agent": "Microsoft Azure Storage Explorer, 1.22.1, darwin azsdk-js-data-tables/13.0.1 core-rest-pipeline/1.3.3 Node/v14.16.0 OS/(x64-Darwin-21.2.0)",
        "x-ms-client-request-id": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
        "x-ms-date": "Fri, 11 Feb 2022 16:56:49 GMT",
        "authorization": "SharedKeyLite REDACTED_STORAGE_ACCOUINT_NAME:ut2uNqtJMqXoQVIZG8pqyrnTUleLsGaSqgAnRAtgc0Q="
      },
      "method": "GET",
      "timeout": 0,
      "disableKeepAlive": false,
      "streamResponseStatusCodes": {},
      "withCredentials": false,
      "tracingOptions": {
        "spanOptions": {},
        "tracingContext": {
          "_currentContext": {}
        }
      },
      "requestId": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
      "allowInsecureConnection": true
    },
    "bodyAsText": "{\"odata.error\":{\"code\":\"InvalidInput\",\"message\":{\"lang\":\"en-US\",\"value\":\"One of the request inputs is not valid.\\nRequestId:3a117790-e002-007b-4768-1f5538000000\\nTime:2022-02-11T16:56:50.0462714Z\"}}}",
    "parsedBody": {
      "odataError": {
        "code": "InvalidInput",
        "message": {
          "lang": "en-US",
          "value": "One of the request inputs is not valid.\nRequestId:3a117790-e002-007b-4768-1f5538000000\nTime:2022-02-11T16:56:50.0462714Z"
        }
      }
    },
    "parsedHeaders": {
      "errorCode": "InvalidInput",
      "content-length": "199",
      "content-type": "application/json",
      "server": "Windows-Azure-Table/1.0 Microsoft-HTTPAPI/2.0",
      "x-ms-request-id": "3a117790-e002-007b-4768-1f5538000000",
      "x-ms-client-request-id": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
      "x-ms-version": "2019-02-02",
      "date": "Fri, 11 Feb 2022 16:56:49 GMT"
    }
  },
  "details": {
    "errorCode": "InvalidInput",
    "content-length": "199",
    "content-type": "application/json",
    "server": "Windows-Azure-Table/1.0 Microsoft-HTTPAPI/2.0",
    "x-ms-request-id": "3a117790-e002-007b-4768-1f5538000000",
    "x-ms-client-request-id": "8864731e-df21-4f2a-bd14-d2db6f0b8df0",
    "x-ms-version": "2019-02-02",
    "date": "Fri, 11 Feb 2022 16:56:49 GMT",
    "odataError": {
      "code": "InvalidInput",
      "message": {
        "lang": "en-US",
        "value": "One of the request inputs is not valid.\nRequestId:3a117790-e002-007b-4768-1f5538000000\nTime:2022-02-11T16:56:50.0462714Z"
      }
    }
  }
}
```

The data inside the Azure Table storage contains 9 rows as shown below.

```csv
PartitionKey,RowKey,Timestamp,HasStarted,HasStarted@type,RowCount,RowCount@type,AnotherTimestamp,AnotherTimestamp@type
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_1,2022-02-11T09:48:30.0044005Z,true,Boolean,3645476,Double,2022-02-09T11:20:09.333869Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_2,2022-02-11T11:54:04.5552271Z,false,Boolean,522701,Double,2022-02-10T17:02:04.085866Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_3,2022-02-11T17:05:10.9764857Z,true,Boolean,1004000,Double,2020-01-01T00:00:00Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_4,2022-02-11T17:05:15.3310399Z,true,Boolean,2586000,Double,2020-01-01T00:00:00Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_5,2022-02-11T17:05:16.6293117Z,true,Boolean,2043000,Double,2020-01-01T00:00:00Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_6,2022-02-11T17:04:26.1307186Z,true,Boolean,3080000,Double,1900-01-01T00:00:00Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_7,2022-02-11T10:35:44.3878892Z,false,Boolean,0,Double,1970-01-01T00:00:00Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_8,2022-02-11T09:48:44.3583491Z,true,Boolean,14172445,Double,2022-02-09T11:08:49.941807Z,DateTime
REDACTED_PARITITION_KEY,REDACTED_ROW_KEY_9,2022-02-11T09:47:02.1387807Z,true,Boolean,480,Double,2022-01-18T08:17:07.666225Z,DateTime
```

### Steps to Reproduce

1. Open Microsoft Azure Storage Explorer.
2. View the Azure Table storage.

### Actual Experience

1. Open Microsoft Azure Storage Explorer.
2. View the Azure Table storage.
3. Data cannot be displayed in the Azure Table storage.
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/305414/153637516-79994615-ea16-4cd8-addd-f5f19b92fe82.png">

### Expected Experience

1. Open Microsoft Azure Storage Explorer.
2. View the Azure Table storage.
3. Data should be displayed in the Azure Table storage.
<img width="381" alt="image" src="https://user-images.githubusercontent.com/305414/153637694-33ea5080-8252-47ca-bd23-2cd175933d68.png">

**Comment by rokkuci** [comment]
Same issue for me.

**Comment by craxal** [comment]
@sheeeng I created a table using the same data that you shared, and I don't seem to be having a problem with it on my end. Would you like to try using a test build to see if your problem is resolved?

https://cralvordtest02.blob.core.windows.net/customer-share/internal/1.23.0-table-rbac/StorageExplorer-darwin-x64.zip?sv=2020-10-02&st=2022-02-14T19%3A38%3A47Z&se=2022-02-21T19%3A38%3A00Z&sr=b&sp=r&sig=7MRYSOgAwO%2FutNgUaApjkzsv0NOPupvImedJ43QxQg0%3D

If you are still encountering issues, please share with us your app logs. Also, consider opening a support ticket with Azure and provide them the request IDs from the errors you're seeing.

**Comment by iangledhill** [comment]
I am seeing the same thing on Win11.
![image](https://user-images.githubusercontent.com/1924873/154017200-3679e2ce-afd6-4c64-84a1-a9061d5635c9.png)

When I query or simply refresh the table I see this error:
 
`{"odata.error":{"code":"InvalidInput","message":{"lang":"en-US","value":"One of the request inputs is not valid.\nRequestId:40a53edf-a002-009d-4040-2243a0000000\nTime:2022-02-15T07:49:34.4367441Z"}}}`

The error in dev tools is shown as:
![image](https://user-images.githubusercontent.com/1924873/154017056-00872fb2-d019-448d-b989-512605de3926.png)

This specific table was working, and other tables in the same storage account work as expected when viewed in storage explorer. 

I get exactly the same experience running the same version of SE on  #Win10.

**Comment by iangledhill** [comment]
Further info: SE 1.21.3 is the most recent version that works correctly with this table.

![image](https://user-images.githubusercontent.com/1924873/154027325-7a001d23-b4c7-4f0c-a0a6-b0be4dda5f2b.png)

**Comment by craxal** [comment]
@iangledhill Same question as above: would you like to try out this test build to see if your issue is resolved? We've resolved a number of table-related bugs, so it would be very helpful to see if those fixes cover this situation.

https://cralvordtest02.blob.core.windows.net/customer-share/internal/1.23.0-table-rbac/StorageExplorer-darwin-x64.zip?sv=2020-10-02&st=2022-02-14T19%3A38%3A47Z&se=2022-02-21T19%3A38%3A00Z&sr=b&sp=r&sig=7MRYSOgAwO%2FutNgUaApjkzsv0NOPupvImedJ43QxQg0%3D

If the problem seems to affect only this specific table, you may also consider submitting a support request to Azure with a the error and the request ID. They'd be able to determine which request input was invalid.

**Comment by iangledhill** [comment]
Thanks Craig - I did look at the 1.23.0 ZIP but it looks like it only contains MacOS?

**Comment by craxal** [comment]
That is correct. I provided a macOS link, since this issue was report on macOS. If you need a test build for other platforms, you can use these links here:
* [Windows](https://cralvordtest02.blob.core.windows.net/customer-share/internal/1.23.0-table-rbac/StorageExplorer-ia32.exe?sv=2020-10-02&st=2022-02-16T17%3A35%3A13Z&se=2022-02-21T17%3A35%3A00Z&sr=b&sp=r&sig=ePsc%2BVSOdkhJ7tY7H9VEud7yykor7PnBEp7kTfOoKVI%3D)
* [macOS](https://cralvordtest02.blob.core.windows.net/customer-share/internal/1.23.0-table-rbac/StorageExplorer-darwin-x64.zip?sv=2020-10-02&st=2022-02-14T19%3A38%3A47Z&se=2022-02-21T19%3A38%3A00Z&sr=b&sp=r&sig=7MRYSOgAwO%2FutNgUaApjkzsv0NOPupvImedJ43QxQg0%3D)
* [Linux (Snap)](https://cralvordtest02.blob.core.windows.net/customer-share/internal/1.23.0-table-rbac/StorageExplorer_latest_amd64.snap?sv=2020-10-02&st=2022-02-16T17%3A36%3A27Z&se=2022-02-21T17%3A36%3A00Z&sr=b&sp=r&sig=a8A2C95F3u5kS1jkoAEAj%2FA0FlpeUpAbVqgm%2FWDGGB8%3D)
* [Linux (.tar.gz)](https://cralvordtest02.blob.core.windows.net/customer-share/internal/1.23.0-table-rbac/StorageExplorer-linux-x64.tar.gz?sv=2020-10-02&st=2022-02-16T17%3A35%3A34Z&se=2022-02-21T17%3A35%3A00Z&sr=b&sp=r&sig=1PpAUHd%2F3uQmiVzm1FT8Tv149qc7nGaIKw7xX4mYN1c%3D)

**Comment by craxal** [comment]
@sheeeng @iangledhill Have you had a chance to give the test build a try?

**Comment by crisismaple** [comment]
I have the same problem with Linux (Snap) 1.23.0, I also tried to create a copy of that table with access issue, same problem occurs in both SE client and SE in azure portal.


{
  "name": "RestError",
  "message": "{\"odata.error\":{\"code\":\"InvalidInput\",\"message\":{\"lang\":\"en-US\",\"value\":\"One of the request inputs is not valid.\\nRequestId:e999b082-6002-0035-592a-30c029000000\\nTime:2022-03-05T00:49:03.0892881Z\"}}}",
  "stack": "RestError: {\"odata.error\":{\"code\":\"InvalidInput\",\"message\":{\"lang\":\"en-US\",\"value\":\"One of the request inputs is not valid.\\nRequestId:e999b082-6002-0035-592a-30c029000000\\nTime:2022-03-05T00:49:03.0892881Z\"}}}\n    at handleErrorResponse (/snap/storage-explorer/32/resources/app/node_modules/@azure/core-client/dist/index.js:1264:19)\n    at deserializeResponseBody (/snap/storage-explorer/32/resources/app/node_modules/@azure/core-client/dist/index.js:1199:45)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
  "statusCode": 400,
  "request": {
    "url": "https://kaching.table.core.windows.net/servergroups()?NextPartitionKey=1!16!c2VydmVyLWdyb3Vw&NextRowKey=",
    "headers": {
      "accept": "application/json;odata=minimalmetadata",
      "x-ms-version": "2019-02-02",
      "dataserviceversion": "3.0",
      "accept-encoding": "gzip,deflate",
      "user-agent": "Microsoft Azure Storage Explorer, 1.23.0, linux azsdk-js-data-tables/13.0.1 core-rest-pipeline/1.5.0 Node/v16.9.1 OS/(x64-Linux-5.13.0-30-generic)",
      "x-ms-client-request-id": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
      "x-ms-date": "Sat, 05 Mar 2022 00:49:02 GMT",
      "authorization": "SharedKeyLite kaching:X9ziDszvQlJ+36BjAR1ZV9adIHGx0UGSaXOcT8r9Ip0="
    },
    "method": "GET",
    "timeout": 0,
    "disableKeepAlive": false,
    "streamResponseStatusCodes": {},
    "withCredentials": false,
    "tracingOptions": {
      "spanOptions": {},
      "tracingContext": {
        "_currentContext": {}
      }
    },
    "requestId": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
    "allowInsecureConnection": true,
    "enableBrowserStreams": false
  },
  "response": {
    "status": 400,
    "headers": {
      "content-length": "199",
      "content-type": "application/json",
      "server": "Windows-Azure-Table/1.0 Microsoft-HTTPAPI/2.0",
      "x-ms-request-id": "e999b082-6002-0035-592a-30c029000000",
      "x-ms-client-request-id": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
      "x-ms-version": "2019-02-02",
      "x-ms-error-code": "InvalidInput",
      "date": "Sat, 05 Mar 2022 00:49:02 GMT"
    },
    "request": {
      "url": "https://kaching.table.core.windows.net/servergroups()?NextPartitionKey=1!16!c2VydmVyLWdyb3Vw&NextRowKey=",
      "headers": {
        "accept": "application/json;odata=minimalmetadata",
        "x-ms-version": "2019-02-02",
        "dataserviceversion": "3.0",
        "accept-encoding": "gzip,deflate",
        "user-agent": "Microsoft Azure Storage Explorer, 1.23.0, linux azsdk-js-data-tables/13.0.1 core-rest-pipeline/1.5.0 Node/v16.9.1 OS/(x64-Linux-5.13.0-30-generic)",
        "x-ms-client-request-id": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
        "x-ms-date": "Sat, 05 Mar 2022 00:49:02 GMT",
        "authorization": "SharedKeyLite kaching:X9ziDszvQlJ+36BjAR1ZV9adIHGx0UGSaXOcT8r9Ip0="
      },
      "method": "GET",
      "timeout": 0,
      "disableKeepAlive": false,
      "streamResponseStatusCodes": {},
      "withCredentials": false,
      "tracingOptions": {
        "spanOptions": {},
        "tracingContext": {
          "_currentContext": {}
        }
      },
      "requestId": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
      "allowInsecureConnection": true,
      "enableBrowserStreams": false
    },
    "bodyAsText": "{\"odata.error\":{\"code\":\"InvalidInput\",\"message\":{\"lang\":\"en-US\",\"value\":\"One of the request inputs is not valid.\\nRequestId:e999b082-6002-0035-592a-30c029000000\\nTime:2022-03-05T00:49:03.0892881Z\"}}}",
    "parsedBody": {
      "odataError": {
        "code": "InvalidInput",
        "message": {
          "lang": "en-US",
          "value": "One of the request inputs is not valid.\nRequestId:e999b082-6002-0035-592a-30c029000000\nTime:2022-03-05T00:49:03.0892881Z"
        }
      }
    },
    "parsedHeaders": {
      "errorCode": "InvalidInput",
      "content-length": "199",
      "content-type": "application/json",
      "server": "Windows-Azure-Table/1.0 Microsoft-HTTPAPI/2.0",
      "x-ms-request-id": "e999b082-6002-0035-592a-30c029000000",
      "x-ms-client-request-id": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
      "x-ms-version": "2019-02-02",
      "date": "Sat, 05 Mar 2022 00:49:02 GMT"
    }
  },
  "details": {
    "errorCode": "InvalidInput",
    "content-length": "199",
    "content-type": "application/json",
    "server": "Windows-Azure-Table/1.0 Microsoft-HTTPAPI/2.0",
    "x-ms-request-id": "e999b082-6002-0035-592a-30c029000000",
    "x-ms-client-request-id": "fd5fa1ed-48d2-4353-94bf-13c814703c45",
    "x-ms-version": "2019-02-02",
    "date": "Sat, 05 Mar 2022 00:49:02 GMT",
    "odataError": {
      "code": "InvalidInput",
      "message": {
        "lang": "en-US",
        "value": "One of the request inputs is not valid.\nRequestId:e999b082-6002-0035-592a-30c029000000\nTime:2022-03-05T00:49:03.0892881Z"
      }
    }
  }
}

**Comment by craxal** [comment]
@crisismaple If you are encountering this issue in the Azure portal as well, you may want to consider opening a service request with the platform team and provide the request ID. Please note that Storage Explorer in the portal is developed and maintained by a different team; these issues are for the desktop Storage Explorer specifically.

One way we can help narrow down this issue is either you:
- Provide a SAS to your table, if the table does not contain any sensitive information.
- Provide a small, sanitized sample of your data that still reproduces the issue that we can test with.a

**Comment by fredhen** [comment]
Hi @craxal,

I am also experiencing this issue even after updating to v1.23.0 and confirmed similar to @iangledhill that v1.21.3 does not have this issue and other tables in the same storage account does not have this problem. We have informed our teams to downgrade to v1.21.3 for now.

It doesn't seem to have been mentioned previously, but in our case it is specifically Classic storage accounts. Unfortunately these tables contains customer data, so I won't be able to provide you with a SAS for the table. I will try and extract some non-sensitive data and put that in a new table as a test.

Do you still recommend creating a support request with Azure to narrow it down? If so I'll go that route first.

**Comment by ScottHelme** [comment]
I am experiencing the same issue when opening one particular table in our storage account:

```
{"odata.error":{"code":"InvalidInput","message":{"lang":"en-US","value":"One of the request inputs is not valid.\nRequestId:b5fb3e04-XXXX-XXXX-XXXX-382f18000000\nTime:2022-03-15T09:41:27.8331521Z"}}}
```


The table also shows with an error icon in the Quick Access menu:

![image](https://user-images.githubusercontent.com/5352840/158350319-e37c7c6d-9f80-4dac-b20e-719b343715a6.png)


When I try to open the table from the Quick Access menu, it says 'Attached Table Error' and that the table may have been deleted:

![image](https://user-images.githubusercontent.com/5352840/158350490-daebe512-f8c1-4c62-8b93-7e6797523777.png)


If I search for the table and manually navigate to it, the table exists and I get the `InvalidInput` error when opening the table:

![image](https://user-images.githubusercontent.com/5352840/158350932-c607940f-f610-4767-acca-80590d4a5ff9.png)


I can open the table just fine through the Azure Portal with no errors:

![image](https://user-images.githubusercontent.com/5352840/158351311-9706cd58-6da5-4541-811b-396f1b0038e4.png)


I'm running the latest version of Storage Explorer:

![image](https://user-images.githubusercontent.com/5352840/158351417-fa4a4121-c626-4d1d-92e5-10669aa9314c.png)

**Comment by craxal** [comment]
@ScottHelme This is very likely the same issue as #5407. The fact you can open the table in the portal does suggest it might not be a platform issues, which helps narrow it down a bit. A few questions:
* Are some of the tables you can still open without an issue in the same storage account as the problematic one?
* Are there any characteristics of the problematic table that could differentiate it from the ones that are working properly?
* What kind/tier/version of storage account does the problematic table belong to?

**Comment by ScottHelme** [comment]
@craxal 

- I have only found one table that does this. All other tables I've looked at in the same storage account open just fine from either Quick Access or by manually searching for them.
- Not that I'm aware of. It doesn't contain any exotic data and has an overlap of content with some other tables that work just fine.
- I found the following information in the 'Overview' section in the Azure Portal, if you were looking for some other data, let me know what/where and I will happily grab it:
    - Location: West US 
    - Primary Location: West US
    - Disk state: Available
    - Performance: Standard
    - Replication: Locally-redundant storage (LRS)
    - Account kind: Storage (classic)
    - Provisioning state: Succeeded

**Comment by ScottHelme** [comment]
@craxal unfortunately, the table in question does contain sensitive data. I'd be more than happy to jump on a Teams call and walk through some debugging steps in real time and provide feedback as we go. I will drop an email to the address you provided.

**Comment by craxal** [comment]
Hey everyone, I think we've narrowed the problem to an issue in the table SDK we're using. The SDK appears to be using a default empty string value for `NextRowKey`, and is putting this in request URLs. This results in the "invalid parameter" error. The reason this only seems to happen for certain tables may be related to partition boundaries. This would also explain why older versions of Storage Explorer (which used a different version of the SDK), do not have this problem.

I have opened https://github.com/Azure/azure-sdk-for-js/issues/20913 and asked a fix take high priority.

Thanks to @ScottHelme for the debugging session that helped us pinpoint the problem.
